### ISSUE TITLE
Default metrics filters

### ISSUE THREAD

**Issue by soyacz** [scylladb/argus#509]
In perf regression tests, there's plenty of metrics but only few are important for most use cases. Allow client code to decide important metrics, so filters are enabled for them - hiding from start non-important ones and speeding up drawing graphs.

Migrated to Jira: [ARGUS-27](https://scylladb.atlassian.net/browse/ARGUS-27)

[ARGUS-27]: https://scylladb.atlassian.net/browse/ARGUS-27?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by fruch** [comment]
@soyacz 
with the new graph views it could be easy to select on as default, isn't it ?

**Comment by soyacz** [comment]
no, it's separate issue. Key metrics will be useful also for summary widget (currently hardcoded)

**Comment by fruch** [comment]
> no, it's separate issue. Key metrics will be useful also for summary widget (currently hardcoded)

I was reading the headline which talks about filters...

**Comment by soyacz** [comment]
> > no, it's separate issue. Key metrics will be useful also for summary widget (currently hardcoded)
> 
> I was reading the headline which talks about filters...

yes, still  will be useful as default view in graphs shows filters and these could be limited only to important metrics so it loads faster
